homeassistant/components/rest/sensor.py

```diff
@@ -224,7 +237,15 @@ class RestData:
     """Class for handling the data retrieval."""
 
     def __init__(
-        self, method, resource, auth, headers, data, verify_ssl, timeout=DEFAULT_TIMEOUT
+        self,
+        method,
+        resource,
+        auth,
+        headers,
+        data,
+        verify_ssl,
+        timeout=DEFAULT_TIMEOUT,
+        client_certificate_path=DEFAULT_FILE_PATH,
```

Comment:
The default of `dict.get()` is `None`. Thus `DEFAULT_FILE_PATH` is not needed but `client_certificate_path` needs to be re-positioned.

Reply:
should also be fixed with current version